### PR TITLE
update to seismic use case documentation

### PR DESCRIPTION
**Pull request by silviamazzoni**
I am still unable to run the mkdocs locally, but the changes I have made are not significant. I have only added some content.

### CODE REVIEW

File: user-guide/docs/usecases/arduino/usecase.md [hunk 1]
```diff
@@ -3,10 +3,17 @@
...
 
-[Simulation on DS - OpenSees](https://www.designsafe-ci.org/rw/workspace/#!/OpenSees::Simulation){target=_blank}<br/>
-[Jupyter notebooks on DS Juypterhub](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis){target=_blank}<br/>
+### Resources
+A collection of educational notebooks to introduce model parameter calibration and site response analysis using OpenSees in DesignSafe-CI. The example makes use of the following DesignSafe resources: 
+
+* DesignSafe tools used by this project:
+  * [Simulation on DS - OpenSees](https://www.designsafe-ci.org/rw/workspace/#!/OpenSees::Simulation){target=_blank}<br/> 
+  * [Visualization on DS - STKO](https://www.designsafe-ci.org/rw/workspace/#!/stko-express-1.0.0u1#workspace){target=_blank}<br/>
+  * [Jupyter notebooks on DS Juypterhub](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis){target=_blank}<br/>
```
Comment (wesleyboar) [inline user-guide/docs/usecases/arduino/usecase.md:13]: I think these need to be indented two more spaces to be rendered as nesting.

| Before | After |
| - | - |
| arudino BEFORE | ardunio AFTER |

File: user-guide/docs/usecases/mosqueda/usecase.md [hunk 2]
```diff
@@ -3,9 +3,20 @@
...
+### Resources 
+
+* DesignSafe tools used by this project:
+    * [Jupyter notebooks on DS Juypterhub](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis){target=_blank}<br/>
+
+* Jupyter Notebooks on DesignSafe
+    * [Case 1. Shake table performance.ipynb](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products%2FShake%20Table%20Experimental%20Data%2FCase%201.%20Shake%20table%20performance.ipynb){target=_blank}<br/>
+    * [Case 2: Structural Response and System Identification.ipynb](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products%2FShake%20Table%20Experimental%20Data%2FCase%202%3A%20Structural%20Response%20and%20System%20Identification.ipynb){target=_blank}<br/>
+    * Jupyter Notebook for Post-Processing Data: [Case 3: Experimental Workflow.ipynb](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products%2FShake%20Table%20Experimental%20Data%2FCase%203%3A%20Experimental%20Workflow.ipynb){target=_blank}<br/>
```
Comment (wesleyboar) [inline user-guide/docs/usecases/mosqueda/usecase.md:16]: None of these links load (for me) the detail I think you intend them to load. Though the URLs match those copied from the [Data Depot Folder](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products%2FShake%20Table%20Experimental%20Data), they do not bring up the modal. Did the links work as you expected?

| Actual (for me) | Expected (I think) |
| - | - |
| actual | expected |

File: user-guide/docs/usecases/rathje/usecase.md [hunk 3]
```diff
@@ -5,10 +5,19 @@
...
+### Resources 
 
+* DesignSafe tools used by this project:
+    * [Simulation on DS - OpenSees](https://www.designsafe-ci.org/rw/workspace/#!/OpenSees::Simulation){target=_blank}<br/> 
+    * [Visualization on DS - STKO](https://www.designsafe-ci.org/rw/workspace/#!/stko-express-1.0.0u1#workspace){target=_blank}<br/>
+    * [Jupyter notebooks on DS Juypterhub](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis){target=_blank}<br/>
+
+* Jupyter Notebooks on DesignSafe
+    * Jupyter Notebook submitting jobs to OpenSeesMP: [SSI_MainDriver.ipynb](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products%2FOpenSees-STKO%2FSSI_MainDriver.ipynb){target=_blank}<br/>
+    * Jupyter Notebook for Post-Processing Data: [SSI_MainDriver.ipynb](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products%2FOpenSees-STKO%2FExample%20post-processing%20scripts.ipynb){target=_blank}<br/>
```
Comment (wesleyboar) [inline user-guide/docs/usecases/rathje/usecase.md:17]: I worry there is no direct link to open the modal I think you expect.

| | Actual (for me) | Expected (I think) |
| - | - | - |
| `SSI_MainDriver` | main actual | main expected |
| Example | example actual | example expected |

Also, the link would (be expected to) load `Example post-processing scripts.ipynb`, but the text for the link is `SSI_MainDriver.ipynb` (same as the line above).

**Review by wesleyboar — COMMENTED** [review]
I apologize for your trouble running the site "Using Local Environment". I am currently only able to do so via "Using Shared Environment"[^1]. The testing process for contributors will be improved.

I've taken screenshots for you.

| Guide | Before | After |
| - | - | - |
| `mosqueda` | mosqueda BEFORE | mosqueda AFTER |
| `overview` | overview BEFORE | overview AFTER |
| `arudino` | arudino BEFORE | ardunio AFTER |
| `erler-mosqueda` | erler-mosqueda BEFORE | erler-mosqueda AFTER |

Please also review specific issues I found while testing.

[^1]: A misnomer, because it is a shared config for a local env. I will rename it.

**Comment by silviamazzoni** [comment]
Doing the update this way is too much work for both you and me. Do you have instructions on how I can get the makedocs work on my windows desktop? (that is what I meant by local environment).

**Comment by silviamazzoni** [comment]
I will resubmit a new PR based on your feedback here.

File: user-guide/docs/usecases/mosqueda/usecase.md (hunk 2 above)
Comment (wesleyboar) [reply user-guide/docs/usecases/mosqueda/usecase.md:16]: Uncertain. I suspect it is not a feature yet. For now, perhaps reference the name, but do not link to it.

E.g. —

```md
    * Jupyter Notebook for Post-Processing Data: "Case 3: `Experimental Workflow.ipynb`" in [Data Depot Folder](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products%2FShake%20Table%20Experimental%20Data){target=_blank}<br/>
 ```
 
— which should render:

> Jupyter Notebook for Post-Processing Data: "Case 3: `Experimental Workflow.ipynb`" in [Data Depot Folder](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products%2FShake%20Table%20Experimental%20Data)

File: user-guide/docs/usecases/rathje/usecase.md (hunk 3 above)
Comment (wesleyboar) [reply user-guide/docs/usecases/rathje/usecase.md:17]: I offer [the same suggestion as similar problem earlier in review](https://github.com/DesignSafe-CI/DS-User-Guide/pull/16#discussion_r1530917748).

**Comment by wesleyboar** [comment]
@silviamazzoni:

Yes, I have a suggestion for running the docs locally. Try "Using Shared Environment" from the Readme[^1] i.e.

1. Install [Docker Compose](https://docs.docker.com/compose/install/).
2. `cd DS-User-Guide`
3. `make build`
4. `make start`

Again I apologize for the delay and difficulty. A recent repository transition is incomplete in this aspect.

[^1]: I might endeavor to fix "Using Local Environment" or request help to do so, but Docker has proven much more reliable.

—

You are always welcome to make and push changes to any PR you open. No need to close and create a new one. Regardless, I'll jump over to the new one, and ask our team for a means to communicate with you directly; so we can get a local environment set up.

**Comment by wesleyboar** [comment]
> [!IMPORTANT]
> New PR is https://github.com/DesignSafe-CI/DS-User-Guide/pull/17.

**Pull request closed without merging**